File: sample-chat-obj-c/sample-chat/Utils/ChatResources.m
//
//  ChatResources.m
//  sample-chat
//
//  Created by Injoit on 2/25/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

#import "ChatResources.h"

@implementation ChatResources

+ (NSBundle *)resourceBundle {
    return [NSBundle mainBundle];
}

+ (UIImage *)imageNamed:(NSString *)name {
    UIImage *image = [UIImage imageNamed:name
                                inBundle:[self resourceBundle]
           compatibleWithTraitCollection:nil];
    
    return image;
}

+ (UINib *)nibWithNibName:(NSString *)name {
    return [UINib nibWithNibName:name bundle:[self resourceBundle]];
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/Classes/Chat/ChatResources.swift
//
//  ChatResources.swift
//  Swift-ChatViewController
//
//  Created by Injoit on 1/28/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

import UIKit

class ChatResources {
  
  class func resourceBundle() -> Bundle {
    return Bundle.main
  }
  
  class func imageNamed(_ name: String) -> UIImage? {
    
    guard let image = UIImage(named: name, in: self.resourceBundle(), compatibleWith: nil) else {
      return UIImage()
    }
    return image
  }
  
  class func nib(withNibName name: String) -> UINib? {
    return UINib(nibName: name , bundle: Bundle.main)
  }
}
